forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -96,6 +101,7 @@ internal class BTPinCollector(
             try {
                 val forageApiError = ForageApiError.ForageApiErrorMapper.from(forageResponse.toString())
                 val error = forageApiError.errors[0]
+                logger.e("Received an error while submitting capture request to BasisTheory: $error.message")
```

Comment:
include `merchantAccount`,  `paymentRef` and `cardToken` in log?

Reply:
https://github.com/teamforage/forage-android-sdk/pull/42#discussion_r1274193522

should omit cardToken